content/swift/concepts/dictionaries/terms/contains/contains.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.contains'
+Description: 'a Boolean value that indicates whether a dictionary contains a specified key-value pair.'

```

Comment:
``` suggestion
Description: 'A Boolean value that indicates whether a dictionary contains a specified key.'

```
The `contains` method of Swift just checks the existence of the key or key-value pairs depending on the context, not necessarily looking for the existence of a key-value pair only, mostly it seems for the existence of a key.